docs/documentation/custom_modules/add_modules.rst

```diff
@@ -87,6 +87,92 @@ that the loop that will allow to solve it needs usage of the :doc:`Newton solver
 A good way to ensure it is to build a Group class that will solve the ImplicitComponent with NewtonSolver. This Group
 should be the system you will register in FAST-OAD.
 
+The CycleGroup class
+====================
+FAST-OAD comes with :class:`~fastoad.model_base.openmdao.group.CycleGroup`, a convenience class that allows to define
+groups with inner solvers.
+
+The point is to standardize options that control the usage of solvers, so they can be set easily using the
+:ref:`model_options <configuration-model-options>` feature.
+
+A group that contains solvers can be defined this way:
+
+.. code-block:: python
+
+    import fastoad.api as oad
+
+    class SimpleCycleGroup(
+        oad.CycleGroup,
+    #  A simple subclassing will assume following attributes
+    #    use_solvers_by_default = True,
+    #    default_linear_solver = "om.DirectSolver",
+    #    default_nonlinear_solver = "om.NonlinearBlockGS",
+    #    default_linear_options={},
+    #    default_nonlinear_options={},
```

Comment:
I could show 
```python
class SimpleCycleGroup(oad.CycleGroup):
    ...
```
and tell it is equivalent to:
```python
class SimpleCycleGroup(
        oad.CycleGroup,
        use_solvers_by_default = True,
        default_linear_solver = "om.DirectSolver",
        default_nonlinear_solver = "om.NonlinearBlockGS",
        default_linear_options={},
        default_nonlinear_options={},
    ):
```
It would be more verbose, for sure. Would it be clearer?

Reply:
Would it be less verbose to show

```python
class SimpleCycleGroup(oad.CycleGroup):
    ...
```

And then say in plain text what it assumes ?